### PR TITLE
UCS/ucx_info: add new Intel CPU models

### PR DESCRIPTION
**Pull request by artpol84**

### CODE REVIEW

**Comment by artpol84** [comment]
based on https://en.wikichip.org/wiki/intel/cpuid

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/7390/ for details (Mellanox internal link).

File: src/ucs/arch/cpu.h [hunk 1]
```diff
@@ -22,6 +22,9 @@ typedef enum ucs_cpu_model {
     UCS_CPU_MODEL_INTEL_SANDYBRIDGE,
     UCS_CPU_MODEL_INTEL_NEHALEM,
     UCS_CPU_MODEL_INTEL_WESTMERE,
+    UCS_CPU_MODEL_INTEL_HASW,
```
Comment (yosefe) [inline src/ucs/arch/cpu.h]: why not use the full (long name)?

**Comment by artpol84** [comment]
@yosefe, fixed

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/7399/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/7400/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/7401/ for details (Mellanox internal link).

**Comment by yosefe** [comment]
bot:bgate:retest

**Pull request merged**